src/components/Toast/README.md

```diff
@@ -68,18 +68,26 @@ class EmbeddedAppToastExample extends React.Component {
 
 Toast should:
 
-- Be used for short messages to confirm an action. Maximum of 2 lines of text.
-- Not be used for actionable links or messages.
-- Not be used for error messages.
-- Be displayed once at the time. If you need multiple toasts, queue them.
+- Be used for short messages to confirm an action.
+- Not go over 3 words.
+- Rarely be used for error messages.
```

Comment:
```suggestion
- Rarely be used for error messages
```

Reply:
We don't punctuate lists unless one of the items in the list is two sentences (then alllll the list is punctuated)